src/main.scss

```
@@ -0,0 +1,40 @@
+body {
+  margin: 0;
+  font-family: Arial;
+  font-size: 100px;
+  color: white;
+}
+
+.container {
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr;
+  grid-template-rows: auto;
+}
+
+.container__box {
+  display: grid;
+  justify-items: center;
+  align-items: center;
+  height: 300px;
+  min-width: 300px;
+}
+
+@for $i from 0 to 6 {
+  .box-#{$i} {
+    background-color: rgb(51*$i, 0, 0);
+  }
+}
+
+@media screen and (max-width: 899px) {
+  .container{
+    grid-template-columns: 1fr 1fr;
```

если больше двух, то:
```suggestion
    grid-template-columns: repeat(2, 1fr);
```

---

ну тут по чарам в тебе більше строка займає